store/file/square.go

```diff
@@ -0,0 +1,131 @@
+package file
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io"
+
+	"golang.org/x/sync/errgroup"
+
+	"github.com/celestiaorg/rsmt2d"
+
+	"github.com/celestiaorg/celestia-node/share"
+)
+
+type square [][]share.Share
```

Comment:
Is this PR used only for ODS? Should we call it so, then? Or will it be used for EDS as well?

Reply:
Or at least the `readSquare` func because its hardcoded to read the ODS, but square mean both ODS and EDS. Need to clarify

Reply:
Sync: call it a `quadrant`